src/images/controlled_access.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's a minor thing but these svgs could be optimized. Running this file through https://github.com/svg/svgo (with prettify) I get
```
<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
  <path d="M8 0c.147 0 .288.032.419.091l5.884 2.496c.688.292 1.203.97 1.197 1.76-.016 3.128-1.29 8.8-6.675 11.378a1.93 1.93 0 0 1-1.65 0C1.789 13.147.515 7.475.472 4.347c.024-.79.538-1.468 1.225-1.76L7.584.091A.993.993 0 0 1 8 0Zm0 13.9c4.284-2.087 5.472-6.71 5.5-9.481L8 2.087V13.9Z" fill="#354052" fill-rule="nonzero"/>
</svg>
```

This is the tool that terra-ui uses to optimize its svgs.